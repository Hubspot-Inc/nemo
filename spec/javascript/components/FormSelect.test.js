import React from "react";
import renderer from "react-test-renderer";

import Component from "../../../app/javascript/components/FormSelect";

const defaultProps = {
  id: "1",
  options: [
    { id: "1", name: "First" },
    { id: "2", name: "Second" },
  ],
};

it("renders as expected", () => {
  const tree = renderer.create(<Component {...defaultProps} />);
  expect(tree.toJSON()).toMatchSnapshot();
});
