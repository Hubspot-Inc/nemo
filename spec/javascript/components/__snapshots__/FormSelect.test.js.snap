// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<select
  className="form-control"
  defaultValue={null}
  id="1"
  name={null}
>
  <option
    value=""
  >
    
  </option>
  <option
    value="1"
  >
    First
  </option>
  <option
    value="2"
  >
    Second
  </option>
</select>
`;
